app/models/pillar.rb

```diff
@@ -1,36 +1,5 @@
 # frozen_string_literal: true
 # Pillar represents a pillar value on Salt.
-#
-# This can override already existing pillar values, or create completely new ones.
-#
-# For example, given that we have on our static pillar:
-#
-#   certificate_information:
-#     subject_properties:
-#       C: DE
-#       O: SUSE
-#
-# We could override them by creating the following Pillar records:
-#
-#   Pillar.create pillar: "certificate_information:subject_properties:C", value: "FR"
-#   Pillar.create pillar: "certificate_information:subject_properties:O", value: "My Company"
-#
-# Lists can be represented as collisions. So, creating the following Pillar records:
-#
-#   Pillar.create pillar: "my_service:extra_ips", value: "127.0.0.1"
-#   Pillar.create pillar: "my_service:extra_ips", value: "127.0.0.2"
-#
-# Would match the following YAML representation:
-#
-#   my_service:
-#     extra_ips:
-#       - 127.0.0.1
-#       - 127.0.0.2
-#
-# This is because how we configure our salt master using this table as an external pillar. For
-# further information you can visit:
-#   - https://github.com/kubic-project/velum/blob/master/docs/salt.md
-#   - https://github.com/kubic-project/salt/blob/master/config/master.d/50-returner.conf
```

Comment:
Why is all this documentation removed?

Reply:
The idea is that from now on we are not tied to the pillar model anymore, and the pillar API controller can render now information from other models. The whole idea of collisions as lists were just a way to represent certain structures on plain mysql information with keys because of how salt uses it, but for example you cannot represent a unitary list in any way (it would be a scalar).

Personally I'd like to see the Pillar model dying so we start to use more specific models (maybe a general configuration model still makes sense), but we are not tied to it anymore.

I can bring back this documentation anyways, but since the lists implementation was actually broken (you cannot represent a list of whatever size since a unitary list is not representable) I don't think collisions will work as expected right now (would have to test it).